### ISSUE TITLE
Incorrect Timestamp formation from Millis in version 4.9.9.

### ISSUE THREAD

**Issue by siarheidudko** [googleapis/nodejs-firestore#1477]
#### Environment details

  - OS: MacOS Big Sur 11.2.3
  - Node.js version: v12.19.0
  - npm version: 7.7.6
  - `@google-cloud/firestore` version: **4.9.9** (in 4.9.8 it works correctly)
  - `firebase-admin` version: 9.6.0

#### Steps to reproduce

  1.  Create Millis `const ms = Date.now();  //  example: 1618516349297`
  2. Create Timestamp from Millis `const timestamp = admin.firestore.Timestamp.fromMillis(ms);  //  Timestamp { _seconds: 1618516349, _nanoseconds: 296999936 }`
  3. Create Millis from Timestamp `const ms2 = timestamp.toMillis();  //  1618516349296`
  4. The result does not match the original one due to incorrectly formed nanoseconds inside the Timestamp.
  5. Same for 4.9.8: `1618516349297 -> Timestamp { _seconds: 1618516349, _nanoseconds: 297000000 } -> 1618516349297`

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by siarheidudko** [comment]
```
const ms = 1618516349297;
const timestamp = require('@google-cloud/firestore').Timestamp.fromMillis(ms);
const ms2 = timestamp.toMillis(); 
console.log(ms === ms2, ms, timestamp, ms2);
//  false 1618516349297 Timestamp { _seconds: 1618516349, _nanoseconds: 296999936 } 1618516349296
```

**Comment by thebrianchen** [comment]
@siarheidudko Sorry about that! This should be fixed in #1468, which should go in with the next release.

**Comment by thebrianchen** [comment]
Closing this ticket since it should be fixed in the 4.10.0 release.
